<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editRoom</class>
 <widget class="QDialog" name="editRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>730</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorOnRoomName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: red;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set new number of beds </string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="roomNewName">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Set new name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="roomNewDescription">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>310</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="markdown">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Set new room description</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorOnRoomCapacity">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: red;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorOnRoomDescription">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{color: red;}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_room">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>640</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Edit room</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>510</y>
     <width>171</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Set room new status</string>
   </property>
  </widget>
  <widget class="QListWidget" name="statusList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>available</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>unavailable</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
